<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>821</width>
    <height>585</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>821</width>
    <height>585</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>数字化农业机械实时导航系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;楷体&quot;;



</string>
  </property>
  <widget class="QGraphicsView" name="imgView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>330</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="gray_check">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>灰度图</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>拍摄现场</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>81</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>工具箱</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>航向偏差</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>横向偏差</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>66</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="heng_err">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">font: italic 10pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>cm</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>130</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="hang_err">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">font: italic 10pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>°</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>处理时间</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>191</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="runTime">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="styleSheet">
        <string notr="true">font: italic 10pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="cropView">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>330</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>作物处理</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>串口通讯助手</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>接收区域</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>190</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>串口号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="serialPort">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="badeRate">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>9600</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="openSerial">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="clearData">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>清除数据</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="closeSerial">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>关闭串口</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Send_Rec_data">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>发送数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit_read">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'楷体'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;接收数据...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_write">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>发送数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="user_online">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>60</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/icon.ico);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="calculate_position">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>80</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>开始计算</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="u4_word">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>70</y>
      <width>36</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>31</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 7pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>ROW：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 7pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>U4：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>31</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 7pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>COL：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pixes_col">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>31</y>
      <width>36</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="w4_word">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>36</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 7pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>W4：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pixes_row">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>31</y>
      <width>36</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>计算世界坐标</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>当前登录</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="openImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>323</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>347</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>559</y>
     <width>761</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 9pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>当前时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="HourMin">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 9pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="date">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 9pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 9pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>开发者：千里小马哥  Mobile：15291836200   Email：15291836200@163.com</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>320</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>打开视频</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>关闭视频</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="writeImg">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>保存屏幕</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopVedio">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>视频暂停</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>523</x>
     <y>320</y>
     <width>254</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="vertical_project">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>投影</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="point_scan">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>线扫</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cropBtn">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>轮廓</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_8</zorder>
  <zorder>groupBox_7</zorder>
  <zorder>groupBox_6</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox</zorder>
  <zorder>imgView</zorder>
  <zorder>gray_check</zorder>
  <zorder>cropView</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>line</zorder>
  <zorder>openImage</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
